test/test_replica_set_client.py

```diff
@@ -343,10 +343,9 @@ def test_connect_with_internal_ips(self):
         with self.assertRaises(AutoReconnect) as context:
             connected(client)
 
-        self.assertEqual(
-            "Could not reach any servers in [('internal-ip', 27017)]."
-            " Replica set is configured with internal hostnames or IPs?",
-            str(context.exception))
+        self.assertTrue(
+            str(context.exception).startswith("Could not reach any servers in [('internal-ip', 27017)]."
+            " Replica set is configured with internal hostnames or IPs?"))
```

Comment:
Can you also wrap this line at 80 (like it was before)? Also it might be nicer to use self.assertIn (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn) because it will give a better error message than assertTrue if/when this assertion fails: `self.assertIn("Could ...", str(context.exception))`